eclass/ninja-utils.eclass

```diff
@@ -56,14 +54,14 @@ _NINJA_UTILS_ECLASS=1
 inherit multiprocessing
 
 case "${NINJA}" in
-	ninja)
-		NINJA_DEPEND=">=dev-util/ninja-1.8.2"
+	ninja-reference)
+		NINJA_DEPEND=">=dev-util/ninja-1.11.1-r3"
 	;;
 	samu)
 		NINJA_DEPEND="dev-util/samurai"
 	;;
 	*)
-		NINJA_DEPEND=""
+		NINJA_DEPEND="|| ( app-alternatives/ninja <dev-util/ninja-1.11.1-r3 )"
 	;;
 esac
```

Comment:
We could simplify this and just always depend on app-alternatives/ninja. If a package has to set one explicitly, it can set both the env variable and its own BDEPEND. If the user sets one explicitly, that's going to be handled in make.conf and preserved automatically.

Reply:
Alternatively, just wait until app-alternatives/ninja is stable before pushing the eclass change.

Reply:
> Alternatively, just wait until app-alternatives/ninja is stable before pushing the eclass change.

That's probably the best course of action.